packages/now-cli/src/types.ts

```diff
@@ -194,7 +194,7 @@ export type DNSRecord = {
   updated: number;
 };
```

Comment:
Is the type `DNSRecord` used? Perhaps it should be deleted.

Reply:
It is. I believe it is used on API responses.
`DNSRecord` => Receiving data from our API
`DNSRecordData` => Parsed record.

Since this is legacy, should we take a different approach to this as well?

Reply:
I'm not sure about legacy. I'll add @mglagola as a reviewer who should know how DNS APIs work.